client/rest/src/main/java/org/elasticsearch/client/RestClient.java

```diff
@@ -119,6 +120,32 @@
         setNodes(nodes);
     }
 
+    /**
+     * Returns a new {@link RestClientBuilder} to help with {@link RestClient} creation.
+     * Creates a new builder instance and sets the nodes that the client will send requests to.
```

Comment:
I think adding an example of what a cloudId looks like to the javadoc would help for people who don't know what that is.